platform/darwin/src/MGLConversionValue.h

```diff
@@ -0,0 +1,68 @@
+#include <string>
+
+class MGLConversionValue {
```

Comment:
@jfirebaugh hah -- yeah, this is exactly the question I started asking myself this morning: https://github.com/mapbox/mapbox-gl-native/pull/8026#discussion_r100799109.  I started out with this wrapper because I didn't know at the time that a fully/cleanly `NSObject`-based hierarchy would be possible.  If you agree that the wrapper's unnecessary, I'd love to remove it.